src/manual-multiple-soulmates/web/index.html

```diff
@@ -56,46 +56,56 @@ <h2 class="blink__anchor advert__title">[%Offer3Title%]</h2>
                     </span>
                 </div>
             </a>
+            <a class="blink advert advert--manual advert--soulmate hide-until-tablet" href="%%CLICK_URL_UNESC%%[%Offer4URL%]" data-link-name="Offer 4 | [%Offer4Title%]" target="_top">
```

Comment:
Instead of having two separate templates I've changed the `NumberOfCards` field to allow 2 cards / 3 cards / 4 cards / 3 cards with form.